### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by asottile-sentry**
pipeline linting has been flaky due to yaml plugin download -- let's cache it all

### CODE REVIEW

File: bin/setup-gocd-cli [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import hashlib
+import io
+import os.path
+import secrets
+import stat
+import subprocess
+import urllib.request
+
+URL = 'https://sentry-dev-infra-assets.storage.googleapis.com/gocd-085ab00-linux-amd64'  # noqa: E501
+SHA256 = '11d517c0c0058d1204294d01bfac987c0eaf9e37ba533ad54107b0949403321e'
```
Comment (mattgauntseo-sentry) [inline bin/setup-gocd-cli:14]: Can you add a comment on how you generated this value (i.e. `Run sha256sum gocd-...-linux-amd64`)

**Comment by mattgauntseo-sentry** [comment]
LGTM - should we just make this action do the linting? Take a path as input and lint the yaml files in that dir

**Review by mattgauntseo-sentry — APPROVED** [review]

File: bin/setup-gocd-cli (hunk 1 above)
Comment (asottile-sentry) [reply bin/setup-gocd-cli:14]: I... copied it out of the shell script, lol

**Comment by asottile-sentry** [comment]
> LGTM - should we just make this action do the linting? Take a path as input and lint the yaml files in that dir

maybe!  for now I just want to get this to be more reliable

**Pull request merged**